kubernetes/types.go

```diff
@@ -494,8 +494,8 @@ type istioResponse struct {
 
 // GenericIstioObject is a type to test Istio types defined by Istio as a Kubernetes extension.
 type GenericIstioObject struct {
-	meta_v1.TypeMeta   `json:",inline"`
-	meta_v1.ObjectMeta `json:"metadata"`
+	meta_v1.TypeMeta   `json:",inline" yaml:",inline"`
+	meta_v1.ObjectMeta `json:"metadata" yaml:"metadata"`
```

Comment:
Nice.
I guess is harmless to the json parsing, isn't it ?
I understand it only adds support for parsing on yaml.